src-tools/CrosshairComponent.cpp

```diff
@@ -237,6 +237,25 @@ void dm::CrosshairComponent::mouseExit(const MouseEvent & event)
 
 void dm::CrosshairComponent::mouseDown(const MouseEvent & event)
 {
+	// If SHIFT is down, activate "panning" mode instead of normal logic.
+	if (event.mods.isShiftDown())
```

Comment:
Unfortunately, SHIFT+drag already has functionality in DarkMark, and has had it for several years now.  The code in your PR completely breaks that feature.

This will need to be assigned to a different key.  I'll look at what can be done.